omod/src/test/java/org/openmrs/module/appointments/web/mapper/AppointmentMapperTest.java

```diff
@@ -229,6 +231,22 @@ public void shouldMapExistingAppointmentWhenPayloadHasAppointmentStatus() throws
 
     }
 
+    @Test
+    public void shouldMapExistingAppointmentWhenPayloadHasAppointmentPriority() throws ParseException {
+        AppointmentRequest appointmentRequest = createAppointmentRequest();
+        appointmentRequest.setPriority("Routine");
```

Comment:
Why are getters/setters for Appointment Priority using String, instead of Enum?

Reply:
Since we referred appointment status, we followed the same for priority as well

Reply:
See comments above